source/_posts/2022-01-07-android-quest.markdown

```diff
@@ -0,0 +1,63 @@
+
+There are over 40 sensors to include in your automations, with plans to include more sensors as we find more data to send back to your Home Assistant server. Ever play on your headset until it dies and then sit around waiting for it to charge up? Using the battery sensors you can get notified when your headset is fully charged and ready to play that next round. Keep the music on in the background while you are playing? Automate your media player volume level based on your microphone being muted. Its not all about sensors either, you may be wondering who rang the doorbell and if its important enough to take off the headset. Just open up the app and you'll be able to see who is there and even talk back if your device supports it!
+
+We are just getting started with things but as we have discovered our users are really good at finding more unique use cases that we didn't even consider. So get your device [setup with Sidequest], [install the app] and start automating today!
+
+## 2022.1 Release
+
+Well we can't just make a post like this and ignore the new release that just went out the other day without highlighting it right?! While the 2022.1 release may not be as jam packed as 2021.12 we do still have some neat features to cover.
+
+*  A new sensor to determine if your devices [work profile] is active by [zmarties]. This sensor will update as soon as a state change is detected allowing you to make unique automations based on the profile being active.
+*  [dshokouhi] has added a new sensor allowing you to see what the [Last Used App] was. Pair this new sensor with the Quest and you can make specific automations based on what game you are currently playing!
+
+![Screenshot of Last Used App](/images/blog/2022-01-android/last_used_app.png)
+
+*  [jpelgrom] has worked on our websocket implementation to finally bring in areas to the app. Device controls is the first feature to get area support so you can easily add entities from areas. The area will also be shown in the device control button itself. Be on the lookout for more area support in other parts of the app as it should make finding entities a lot easier.
+*  [JBassett] has also made more progress on the websocket implementation by bringing in better error handling to prevent the application from crashing and producing a lot of unneccessary errors.
```

Comment:
```suggestion
*  [JBassett] has also made more progress on the WebSocket implementation by bringing in better error handling to prevent the application from crashing and producing a lot of unnecessary errors.
```

Reply:
Thank you for all of the suggestions :)

Reply:
Oh and the speedy review 😃 !!